lib/es-models/bib.js

```diff
@@ -123,6 +138,52 @@ class EsBib extends EsBase {
     if (dateBy008) return dateBy008
   }
 
+  // Returns array of hashes with { url, label }, containing electronic resources that are complete digital records
+  electronicResources () {
+    const resources = this._extractElectronicResourcesFromBibMarc('complete digital record')
+    return resources?.filter((resource) => !this._isAeonUrl(resource.url)) || null
+  }
+
+  /**
+ * Given a MiJ bib and a e-resource type, returns extracted resources
+ *
+ * @param {object} bib Instance of BibSierraRecord
+ * @param {string} type Type of resource to extract - either 'supplementary content' or 'complete digital record'
+ *
+ * @return {array} Array of objects representing electronic resources with following properties:
+ * - url: URL of resoruce
+ * - label: Label for resource if found
+ * - path: "Path" to value in marc (for recording provo)
```

Comment:
Should remove this `path` note since we're no longer returning it. Also note possible return of `null` in place of empty array